packages/babel-plugin-fbtee/src/bin/collect.tsx

```diff
@@ -80,112 +81,105 @@ export type CollectFbtOutput = {
 
 export type CollectFbtOutputPhrase = CollectFbtOutput['phrases'][number];
 
-const args = {
-  COMMON_STRINGS: 'fbt-common-path',
-  CUSTOM_COLLECTOR: 'custom-collector',
-  GEN_FBT_NODES: 'gen-fbt-nodes',
-  GEN_OUTER_TOKEN_NAME: 'gen-outer-token-name',
-  HASH: 'hash-module',
-  HELP: 'h',
-  MANIFEST: 'manifest',
-  OPTIONS: 'options',
-  PACKAGER: 'packager',
-  PLUGINS: 'plugins',
-  PRESETS: 'presets',
-  PRETTY: 'pretty',
-  TRANSFORM: 'transform',
-} as const;
```

Comment:
This stuff is not needed since yargs is now type-safe with TypeScript.

Reply:
yargs allows you to provide a definition and then it returns a typed object, ie. `args.pretty` will be typed as "boolean".

You are right, for the description etc. though. Might make sense to snapshot the `--help` command.